en/debug/system_wide_replay.md

```diff
@@ -1,93 +1,112 @@
 # System-wide Replay
 
-Based on ORB messages, it's possible to record and replay arbitrary parts of the system.
+It is possible to record and replay arbitrary parts of the system based on ORB messages.
 
 Replay is useful to test the effect of different parameter values based on real data, compare different estimators, etc.
 
 ## Prerequisites
 
-The first thing that needs to be done is to identify the module or modules that should be replayed.
+The first step is to identify the module or modules that should be replayed.
 Then, identify all the inputs to these modules, i.e. subscribed ORB topics.
 For system-wide replay, this consists of all hardware input: sensors, RC input, MAVLink commands and file system.
 
 All identified topics need to be logged at full rate (see [logging](../dev_log/logging.md)).
 For `ekf2` this is already the case with the default set of logged topics.
 
 It is important that all replayed topics contain only a single absolute timestamp, which is the automatically generated field `timestamp`.
-Should there be more timestamps, then they must be relative with respect to the main timestamp.
+Should there be more timestamps, they must be relative with respect to the main timestamp.
 For an example, see [SensorCombined.msg](https://github.com/PX4/PX4-Autopilot/blob/main/msg/SensorCombined.msg).
 Reasons for this are given below.
 
 
 ## Usage
 
-- First, choose the file to replay, and build the target (from within the PX4-Autopilot directory):
+- First, choose the file to replay and build the target (from within the PX4-Autopilot directory):
   
   ```sh
   export replay=<absolute_path_to_log_file.ulg>
   make px4_sitl_default
   ```
   This will create the output in a separate build directory `build/px4_sitl_default_replay` (so that the parameters don't interfere with normal builds).
```

Comment:
```suggestion
  This will create the build/make output in a separate build directory `build/px4_sitl_default_replay` (so that the parameters don't interfere with normal builds).
```